toolchains/install_bootstrap/android/tool_redirect_rules.yaml

```diff
@@ -0,0 +1,30 @@
+bin: /toolchain_root/bin
+
+overwrite: []
+
+replacements:
+  cc:  arm-linux-androideabi-gcc
```

Comment:
What if you set `bin` to `/toolchain_root/bin/arm-linux-androideabi-`, could you then move all replacements to overwrites? I'd probably rename `bin` to `prefix` as it close to the naming scheme used in places like `configure` script (you could in theory also introduce a `program-prefix` field).